docs/source/x_ext.rst

```diff
@@ -434,84 +440,50 @@ Issue interface
-  +------------------------+----------------------+------------------------------------------------------------------------------------------------------------------+
+  +------------------------+------------------------+------------------------------------------------------------------------------------------------------------------+
+  | **Signal**             | **Type**               | **Description**                                                                                                  |
+  +------------------------+------------------------+------------------------------------------------------------------------------------------------------------------+
+  | ``accept``             | logic                  | Is the offloaded instruction (``id``) accepted by the |coprocessor|?                                             |
+  +------------------------+------------------------+------------------------------------------------------------------------------------------------------------------+
+  | ``writeback``          | logic                  | Will the |coprocessor| perform a writeback in the core to ``rd``?                                                |
+  |                        |                        | Writeback to ``X0`` is allowed by the |coprocessor|, but will be ignored by the |processor|.                     |
+  |                        |                        | A |coprocessor| must signal ``writeback`` as 0 for non-accepted instructions.                                    |
+  +------------------------+------------------------+------------------------------------------------------------------------------------------------------------------+
+  | ``dualwrite``          | logic                  | Will the |coprocessor| perform a dual writeback in the core to ``rd`` and ``rd+1``?                              |
+  |                        |                        | Only allowed if ``X_DUALWRITE`` = 1 and instruction bits ``[11:7]`` are even.                                    |
+  |                        |                        | Writeback to the ``X0``, ``X1`` pair is allowed by the |coprocessor|, but will be ignored by the |processor|.    |
+  |                        |                        | A |coprocessor| must signal ``dualwrite`` as 0 for non-accepted instructions.                                    |
+  +------------------------+------------------------+------------------------------------------------------------------------------------------------------------------+
+  | ``register_read``      | :ref:`registerflags_t  | Will the |coprocessor| perform require specific registers to be read?                                            |
+  |                        | <registerflags>`       | A |coprocessor| may only request an odd register of a pair, if it also requests the even pair                    |
```

Comment:
the even pair -> the even register of a pair

Reply:
will fix